content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -111,6 +111,31 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'mssql']}>
+
+1. Set the `provider` and `url` fields of the `datasource` block in your Prisma schema as shown:
+
+   ```prisma file=prisma/schema.prisma
+   datasource db {
+     provider = "sqlserver"
+     url      = env("DATABASE_URL")
+   }
+   ```
```

Comment:
````suggestion
```prisma file=prisma/schema.prisma
datasource db {
 provider = "sqlserver"
 url      = env("DATABASE_URL")
}
```
````
No list and repetition in other databases.